packages/algob/src/lib/compile.ts

```diff
@@ -161,6 +178,18 @@ export class PyCompileOp {
     return pyCompiled;
   }
 
+  /**
+   * Replaces parameters with the values passed
+   * @param program Teal program
+   * @param replaceParams params that needs to be replaced in program
+   */
+  replaceTempValues (program: string, replaceParams: ReplaceParams): string {
+    for (const param in replaceParams) {
+      program = program.split(param).join(replaceParams[param]);
+    }
+    return program;
+  }
```

Comment:
Shall we add test(s) ?

Reply:
i have updated the example to test this functionality

Reply:
let's add mocks also